lib/internal/streams/promises.js

```diff
@@ -4,9 +4,13 @@ const {
   Promise,
 } = primordials;
 
-function pipeline() {
+let pl;
```

Comment:
pipeline

Reply:
There will be a conflict with the function name.